src/CoreShop/Bundle/OrderBundle/Expiration/OrderExpiration.php

```diff
@@ -51,7 +63,15 @@ public function expire($days, $params = [])
 
         if (is_array($orders)) {
             foreach ($orders as $order) {
-                $this->stateMachineApplier->apply($order, OrderTransitions::IDENTIFIER, OrderTransitions::TRANSITION_CANCEL);
+                $this->stateMachineApplier->apply($order, OrderTransitions::IDENTIFIER,
+                    OrderTransitions::TRANSITION_CANCEL);
```

Comment:
Don't know if your code style has a maximum line length, but I think it should either be
```suggestion
                $this->stateMachineApplier->apply($order, OrderTransitions::IDENTIFIER, OrderTransitions::TRANSITION_CANCEL);
```
or 
```suggestion
                $this->stateMachineApplier->apply(
                    $order, 
                    OrderTransitions::IDENTIFIER,
                    OrderTransitions::TRANSITION_CANCEL
                );
```

Reply:
thanks

Reply:
My IDE has a max line length, thats why sometimes things like that happen. But I sometimes run the auto style fixer anway.